### ISSUE TITLE
cannot run zsh: IOT instruction  ./Downloads/Spacedrive-linux-x86_64.AppImage

### ISSUE THREAD

**Issue by newyorkthink** [spacedriveapp/spacedrive#1915]
### Describe the bug

zsh: IOT instruction  ./Downloads/Spacedrive-linux-x86_64.AppImage

### Reproduction

![image](https://github.com/spacedriveapp/spacedrive/assets/87791643/c43f6b61-ed03-45dd-92cb-5dfb44d0bbb7)

### Expected behavior

![image](https://github.com/spacedriveapp/spacedrive/assets/87791643/29e7ad4a-6e2b-4a93-85ae-9e95b01a6bf2)

### Platform and versions

```Shell
zsh: IOT instruction  ./Downloads/Spacedrive-linux-x86_64.AppImage
```

**Comment by brxken128** [comment]
The actual error is the previous line, IOT instruction means that `abort()` was called due to an error within the app I believe - this comes from `FD_SETSIZE`. I assumed this was fixed but I'll enquire with the team! 🙂

**Comment by HeavenVolkoff** [comment]
Yeah, it is already fixed in #1785

**Comment by brxken128** [comment]
I'll close this for now as we already have the fix, we're just waiting on the merge 😄
